### DISCUSSION TITLE
Support for pin code type login?

### DISCUSSION THREAD

**Discussion by bbakkebo** [zitadel/zitadel#4800] [General]
I am new to Zitadel and just wanting to know if there is any thought to support a pin code type login.  I see that documents talk about windows hello, which has a pin code type login.  But I do not see any documentation on how this would be done.
Really was thinking of a simple way that they have to setup multifactor ie email, password and pin code, when they first register and afterwards they could just use email and pin..

**Comment by mffap** [comment]
Hey @bbakkebo - you should try out Passkeys. You can set it up for your account under the path `/ui/console/users/me?id=security`.

![2022-11-30 17_13_47-ZITADEL • Console](https://user-images.githubusercontent.com/1366906/204850369-2e1b8cec-2269-464b-b7a8-df9e491faa5e.png)

Basically the devices prompts you for biometrics, PIN etc. to unlock and then you're logged in. Here an example: 
https://youtu.be/cZjHQYurSjw

We are working on making it easier during registration and login to setup passwordless: 
- https://github.com/zitadel/zitadel/issues/1610
- https://github.com/zitadel/zitadel/issues/1991

For now you can add passwordless via API to users (eg, for custom registration): 
https://docs.zitadel.com/docs/guides/manage/user/reg-create-user#add-passwordless-to-existing-user
